engine/crates/federated-graph/src/federated_graph.rs

```diff
@@ -59,8 +59,7 @@ pub struct Object {
 
     pub implements_interfaces: Vec<InterfaceId>,
 
-    /// All _resolvable_ keys.
-    pub resolvable_keys: Vec<Key>,
+    pub keys: Vec<Key>,
```

Comment:
That would break the JSON format, I'm not up to date on whether this is going to be a problem, wdyt?

Reply:
oh yeah, you're right! forgot about that, good catch!